staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -65,12 +66,19 @@ type queueSet struct {
 
 	lock sync.Mutex
 
-	// config holds the current configuration.  Its DesiredNumQueues
-	// may be less than the current number of queues.  If its
-	// DesiredNumQueues is zero then its other queuing parameters
-	// retain the settings they had when DesiredNumQueues was last
-	// non-zero (if ever).
-	config fq.QueueSetConfig
+	// qCfg holds the current queuing configuration.  Its
+	// DesiredNumQueues may be less than the current number of queues.
+	// If its DesiredNumQueues is zero then its other queuing
+	// parameters retain the settings they had when DesiredNumQueues
+	// was last non-zero (if ever).
+	qCfg fq.QueuingConfig
```

Comment:
An idea is to use embedding to keep the prior name, e.g. elsewhere in the package make:

```
type config struct {
  fq.QueuingConfig
  fq.DispatchingConfig
}
```

I'm not asking for a change since this is private and therefore easy to change any time, I just thought I'd offer an alternative to changing to qCfg/dCfg everywhere.

Reply:
I now have so many copies of this change, I am not going to worry about this.